distribution/lib/Standard/Base/0.0.0-dev/src/Any.enso

```diff
@@ -264,7 +265,8 @@ type Any
 
              "Hello".if_nothing ""
     if_nothing : Any -> Any
-    if_nothing self ~_ = self
+    if_nothing self ~other =
+        const self other
```

Comment:
Feel free to leave as is, but reading this code I actually feel like including the logic for ignoring the value in-line could make this more readable:

```suggestion
    if_nothing self ~other =
        ignore_input ~_=Nothing
        ignore_input other
        self
```

but I guess `const` is all good to as long as you know its definition suspends the argument